src/libraries/System.Numerics.Tensors/src/System/Numerics/Tensors/TensorPrimitives.netstandard.cs

```diff
@@ -311,6 +311,93 @@ private static float MinMaxCore<TMinMaxOperator>(ReadOnlySpan<float> x, TMinMaxO
             return result;
         }
 
+        /// <remarks>
+        /// This is the same as <see cref="Aggregate{TTransformOperator, TAggregationOperator}(ReadOnlySpan{float}, TTransformOperator, TAggregationOperator)"/>
```

Comment:
This comment looks wrong.

Reply:
Yep. Must have been a copy paste error.

Reply:
It is, must have been a copy paste error. Removed.